students/olszewski_bartosz/lesson_01_python_basics/apple_sharing.py

```diff
@@ -0,0 +1,6 @@
+print('Podaj liczbę uczniów')
+a = (input())
+print('Podaj liczbę jabłek')
+b = (input())
+print ('Każdy z uczniów otrzymał',int(b)//int(a))
+print ('W koszyku pozostało',int(b)%int(a))
```

Comment:
PEP 8: missing whitespace around modulo operator

PEP8 Warning in PyCharm is underlined with the orange curvy line. If you hover your mouse pointer over this curvy line, the detailed information about the error appears at the tooltip.
![image](https://user-images.githubusercontent.com/8876350/36839685-07d9f5a2-1d43-11e8-9cee-a8ceefe318b6.png)


Even better soultion, always check you code before PR is created with inspect tool. Just select 'Code' -> 'Inspect Code'  in PyCharm IDE
![image](https://user-images.githubusercontent.com/8876350/36839530-aacd3fae-1d42-11e8-8c35-5580f29b80f4.png)